docs/reference/transfers.md

```diff
@@ -364,11 +366,60 @@ Mark the transfer as a [post-pending transfer](#post-pending-transfer).
 
 Mark the transfer as a [void-pending transfer](#void-pending-transfer).
 
+### `flags.debits_at_most`
+
+Transfer at most [`amount`](#amount) — automatically transferring less than `amount` if necessary
+to obey the debit account's constraints. If the highest amount transferable is `0`, the respective
+"overflow" or "exceeds" result code is returned.
+
+The `amount` of the recorded transfer is set to the actual amount that was transferred, which is
+less than or equal to the amount that was passed to `create_transfers`.
+
+`flags.debits_at_most` is compatible with:
+- `flags.linked`
+- `flags.pending`
+- `flags.credits_at_most`
+
+`flags.debits_at_most` is incompatible with:
+- `flags.post_pending_transfer`
+- `flags.void_pending_transfer`
```

Comment:
Also, rather than making it exclusive, why not implement this by using up to the pending transfer value?

Reply:
I left it deliberately unimplemented/incompatible for now, waiting on a decision on partial-resolution of pending transfers, since that would involve a different implementation here anyway.